pkg/service/node.go

```diff
@@ -340,3 +361,23 @@ func getDeviceByMountPoint(mp string) (string, error) {
 	}
 	return s[1], nil
 }
+
+func getDeviceUsage(devicePath string) ([]int, error) {
```

Comment:
The correct approach is to check the volumes directly on the node as we can't be certain if engine is reporting correct stats in case volume is a block device.

So there are two cases which can occur, as we discussed:

1) volume is backed by RAW or block device
In this case there is no easy way to report real usage of the device, however total capacity can and should still be reported.

2) volume has a filesystem on top
In this case we can report real usage using unix.Statfs if we rule out block device case first.

I'll post new version of the patch for review where this approach is implemented. Other drivers like GCE use the same approach.